### ISSUE TITLE
OBS Often Fails to Launch Scene Collection + Big Crash Log

### ISSUE THREAD

**Issue by proactivematter** [obsproject/obs-studio#7854]
### Operating System Info

Other

### Other OS

MacOS 13.0.1

### OBS Studio Version

28.1.2

### OBS Studio Log URL

https://obsproject.com/logs/CO08P_idH6eCykIU

### OBS Studio Crash Log URL

https://drive.google.com/file/d/1Bx1nwUvRB4D4-gM3dt-0dA97o20REwki/view?usp=sharing

### Expected Behavior

Launch OBS into scene collection

### Current Behavior

OBS hangs on launch and must be Force Quit.

### Steps to Reproduce

N / A

### Anything else we should know?

I seem to have a problem with my main scene collection, which is pretty complex and has a lot of stuff on it. When I try to launch OBS it will just keep bouncing on the Dock, and then I have to Force Quit it. The MacOS error log that results from that is huge and you can [find it here](https://drive.google.com/file/d/1Bx1nwUvRB4D4-gM3dt-0dA97o20REwki/view?usp=sharing).

I've tried moving all Addons out from Application Support, and it still happens so it's not related.

If I move the json for the scene out of the folder, it will default to another scene collection and work fine. If I try to import the scene collection back, it will again fail to load it if I try to switch scene collections.

The weird thing is that it does SOMETIMES launch the scene collection correctly, and when it does everything works fine.

Let me know if there is anything else you need me to provide.

Thanks for the help.

**Comment by proactivematter** [comment]
As an update, I've spent many many hours troubleshooting this, and here's how I found a fix:

- Created test user account on MacOS
- Imported problematic scene without issue!
- Tried cleaning up and reinstalling OBS on my main user account, including deleting application support files, then with a fresh install importing the scene collection. No luck.
- Disabled every startup/background apps thinking there may be a conflict. No luck
- Finally decided to remove scene by scene in the problematic scene collection. Found 1-2 scenes (out of 3) that were using the new MacOS Screen Capture (Beta), and upon deleting and recreating these 1-2 scenes, the issue was resolved.

So I'm not sure what was happening there with those 1-2 scenes. They each only had 3 sources on them - by far my simplest scenes, out of many others. Maybe it had something to do with the new MacOS Screen Capture. Maybe they'd just gotten corrupted for an unrelated reason - if that's even a thing.

I still have the broken scene collection backed up. Maybe it could be useful for solving this weird issue. But for now I'm happy this is resolved and all my hard work on this collection is not lost.
